Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoISE.yaml

```diff
@@ -0,0 +1,68 @@
+    let CiscoISEAuthParser=(disabled:bool=false){
+      Syslog
+      | where not(disabled)
+      | where ProcessName has_any ("CISE", "CSCO")
+      | parse kind = regex SyslogMessage with @"\d{10}\s" EventOriginalType @"\s(NOTICE|INFO|WARN|WARNING|ERROR|FATAL|DEBUG)"
+      | where EventOriginalType in ("25104","25105","25106","25112","51000","51001","51002","51003","51004","51005","51006","51007","51008","51009","51020","51021","51022","51106","60075","60076","60077","60078","60080","60081","60082","60135","60136","60204","60205","60206","61012","61013","61014","61015","60507","51025","61076","61077","10003","10004","10005","10006","10007","22000","22004","22028","22037","22040","22091","5400","5401","5412","5418","5447","5448","86010","86011","86014","86020","24015","24020","24021","24022","24050","24054","24055","24056","24057","24337","24338","24402","24403","24406","24407","24408","24409","24410","24414","24415","24418","24454","24470","24484","24485","24486","24487","24489","24490","24491","24492","24496","24505","24508","24518","24547","24612","24613","24614","24639","24704","24705","24706","24707","24708","24709","24712","24814","24853","24854","24860","24890","24716","24717","24719","89157","89159","89160","33102","33103","5200","5201","5231","11002","11003","11039","11052","11812","11813","11814","11815","11823","11824","12005","12006","12208","12306","12307","12308","12506","12507","12528","12529","12612","12613","12614","12615","12623","12624","12705","12706","12707","12854","12975","12976","11700")
```

Comment:
We don't need to repeat any codes again. Distinct EventOriginalType from the table should be able to get us the list of these.
In fact please filter the table itself in vim parser so only the required codes are fetched.
Ex: for eventresult == "Success", allow only the codes which are present with success as the event result so that prefiltering can be achieved.
Refer to this parser to learn more about it - https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimAuditEvent/Parsers/vimAuditEventMicrosoftWindowsEvents.yaml (line 104 onwards)

Reply:
Not sure how this can be applicable for Authentication filtering parser as we have only options to filter on `starttime`, `endtime`, and `targetusername` fields
Moved all eventids to dynamic list `EventOriginalTypeList`

Reply:
`EventOriginalTypeList` is created from lookup table and then used for filtering